custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
```

Comment:
Consider using meaningful variable names to enhance code readability and maintainability.